### PR TITLE
[PTT-186] configure service secondary percentages

### PR DESCRIPTION
**Pull request by filma80**
#### List of Changes
Secondary percentages (TEFA) are disabled based on installment date.

#### Motivation and Context
REF-P can't insert TEFA for dates before 1/7/2021

#### How Has This Been Tested?
Unit and integration tests.

#### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/app/pages/tributi-page/tributi-step2-definisci-rate/tributi-step2-definisci-rate.component.ts [hunk 1]
```diff
@@ -229,4 +229,15 @@ export class TributiStep2DefinisciRateComponent implements OnInit {
       control?.setValue(Math.round(parseFloat(control.value) * 100 - 1) / 100);
     }
   }
+
+  isPercentageSecondaryDisabled(i: number): boolean {
+    const dueDate = this.installments.controls[i].get('dueDate');
+    const percentageSecondary = this.installments.controls[i].get('percentageSecondary');
+    if (!this.isDateAfter_30_06_2021(dueDate?.value)) {
```
Comment (GiovanniMancini) [inline src/app/pages/tributi-page/tributi-step2-definisci-rate/tributi-step2-definisci-rate.component.ts:236]: This date shouldn't be hardcoded

**Review by GiovanniMancini — CHANGES_REQUESTED** [review]
Please remove hardcoded values

File: src/app/pages/tributi-page/tributi-step2-definisci-rate/tributi-step2-definisci-rate.component.ts (hunk 1 above)
Comment (filma80) [reply src/app/pages/tributi-page/tributi-step2-definisci-rate/tributi-step2-definisci-rate.component.ts:236]: This date is already parameterized in PTT-187

**Comment by filma80** [comment]
This date is already parameterized in PTT-187

**Review by GiovanniMancini — APPROVED** [review]

**Pull request merged**